### PR TITLE
Small PRs

### PR DESCRIPTION
**Pull request by hyd628**
### Description/Original PRs

https://github.com/PureStake/moonbeam-docs/pull/404
https://github.com/PureStake/moonbeam-docs/blame/master/.snippets/text/batch/batch-interface.md#L7-L12
https://github.com/PureStake/moonbeam-docs/pull/393
https://github.com/PureStake/moonbeam-docs/pull/413
https://github.com/PureStake/moonbeam-docs/pull/411

### Checklist

- [x] If this requires removing old images from the `moonbeam-docs` repo, I have created a corresponding PR
- [x] If this requires adding/updating/deleting redirects, I have created a corresponding PR in the `moonbeam-mkdocs` repo
- [x] If this requires removing old variables from the `moonbeam-docs` repo, I have created a corresponding PR

### Corresponding PRs

Please link to any additional corresponding PRs here.

### CODE REVIEW

**Review by eshaben — CHANGES_REQUESTED** [review]
Thanks @hyd628! Just one thing that needs to be reviewed

**Comment by hyd628** [comment]
Thanks @eshaben , incorporated the feedback.

File: builders/pallets-precompiles/precompiles/staking.md [hunk 1]
```diff
@@ -93,16 +93,19 @@ Moonbeam使用一种名为[平行链质押](/builders/pallets-precompiles/pallet
...
  - **executeDelegationRequest**(*address* delegator, *address* candidate) —— 执行任何在提供委托人和候选人地址的情况下的可用委托请求。使用质押Pallet的[`executeDelegationRequest`](/builders/pallets-precompiles/pallets/staking/#:~:text=executeDelegationRequest(delegator, candidate)){target=_blank}函数
  - **cancelDelegationRequest**(*address* candidate) —— 取消提供的候选人地址中任何待处理的委托请求。使用质押Pallet的[`cancelDelegationRequest`](/builders/pallets-precompiles/pallets/staking/#:~:text=cancelDelegationRequest(candidate)){target=_blank}函数
 
-以下的函数已被**弃用**，将会在近期移除：
+从运行时1800开始，以下方法已被**弃用**：
+
+ - **scheduleLeaveDelegators**() —— 发起离开委托人集并撤回所有进行中的委托的请求。发起请求并不会自动执行，需要等待[退出延迟](#exit-delays)，您可以通过`executeLeaveDelegators`函数执行请求。使用质押Pallet的[`scheduleLeaveDelegators`](/builders/pallets-precompiles/pallets/staking/#:~:text=scheduleLeaveDelegators()){target=_blank}函数。可以使用[批量处理方法](/builders/pallets-precompiles/pallets/utility/#using-the-batch-extrinsics){target=_blank}来打包多个`scheduleLeaveDelegators`请求
```
Comment (eshaben) [inline builders/pallets-precompiles/precompiles/staking.md:104]:
```suggestion
 - **scheduleLeaveDelegators**() —— 发起离开委托人集并撤回所有进行中的委托的请求。发起请求并不会自动执行，需要等待[退出延迟](#exit-delays)，您可以通过`executeLeaveDelegators`函数执行请求。可以使用[批量处理方法](/builders/pallets-precompiles/pallets/utility/#using-the-batch-extrinsics){target=_blank}来打包多个`scheduleLeaveDelegators`请求
```

File: builders/pallets-precompiles/precompiles/staking.md [hunk 2]
```diff
@@ -93,16 +93,19 @@ Moonbeam使用一种名为[平行链质押](/builders/pallets-precompiles/pallet
...
+ - **executeLeaveDelegators**(*address* delegator, *uint256* delegatorDelegationCount) —— 执行离开委托人集和撤回所有委托的可用请求。使用质押Pallet的[`executeLeaveDelegators`](/builders/pallets-precompiles/pallets/staking/#:~:text=executeLeaveDelegators(delegator, delegationCount)){target=_blank}函数。可以使用[批量处理方法](/builders/pallets-precompiles/pallets/utility/#using-the-batch-extrinsics){target=_blank}来打包多个`executeLeaveDelegators`请求
```
Comment (eshaben) [inline builders/pallets-precompiles/precompiles/staking.md:105]:
```suggestion
 - **executeLeaveDelegators**(*address* delegator, *uint256* delegatorDelegationCount) —— 执行离开委托人集和撤回所有委托的可用请求。可以使用[批量处理方法](/builders/pallets-precompiles/pallets/utility/#using-the-batch-extrinsics){target=_blank}来打包多个`executeLeaveDelegators`请求
```

File: builders/pallets-precompiles/precompiles/staking.md [hunk 3]
```diff
@@ -93,16 +93,19 @@ Moonbeam使用一种名为[平行链质押](/builders/pallets-precompiles/pallet
...
+ - **cancelLeaveDelegators**() —— 取消待处理的离开委托人集的请求。使用质押Pallet的[`cancelLeaveDelegators`](/builders/pallets-precompiles/pallets/staking/#:~:text=cancelLeaveDelegators()){target=_blank}函数。可以使用[批量处理方法](/builders/pallets-precompiles/pallets/utility/#using-the-batch-extrinsics){target=_blank}来打包多个`cancelLeaveDelegators`请求
```
Comment (eshaben) [inline builders/pallets-precompiles/precompiles/staking.md:106]:
```suggestion
 - **cancelLeaveDelegators**() —— 取消待处理的离开委托人集的请求。可以使用[批量处理方法](/builders/pallets-precompiles/pallets/utility/#using-the-batch-extrinsics){target=_blank}来打包多个`cancelLeaveDelegators`请求
```

**Review by eshaben — APPROVED** [review]
LGTM ✨

**Pull request merged**